Makefile.am

```diff
@@ -0,0 +1,32 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
+ACLOCAL_AMFLAGS = -I m4
+
+AM_CXXFLAGS = -std=c++11 -Wall -Werror
+AM_CPPFLAGS = \
+	-I$(srcdir)/common/json \
+	-I$(srcdir)/vendor/boost_1_80_0
+
+# Two daemons
+bin_PROGRAMS = men-authd men-updated
```

Comment:
Finally could have gotten `menderd` :p

Reply:
@kacf

Reply:
`men-update` -- wow, updating men!

Reply:
I have added this to the next client team meeting agenda on Monday. @lluiscampos: Could you invite @vpodzime to this meeting?